python/ray/tests/test_dynres.py

```diff
@@ -569,3 +569,25 @@ def delete_res(resource_name, res_client_id):
             all_resources_created.append(str(i) in resources)
         success = all(all_resources_created)
     assert success
+
+
+def test_release_cpus_when_actor_creation_task_blocking(shutdown_only):
+    ray.init(num_cpus=2)
+
+    @ray.remote(num_cpus=1)
+    def get_100():
+        import time
+        time.sleep(2)
```

Comment:
nit: We should avoid fixed duration sleeps. Consider having this task wait on a fixed object id, that is put into the object store by the driver/actor as a signalling mechanism (see example in `test_dynamic_res_concurrent_res_increment`).

Reply:
Waiting on a fixed object can't make sure raylet go into `HandleTaskBlocked` for `__init__` task.

Reply:
Can we make the sleep shorter without making the test flaky?